userspace/falco/stats_writer.cpp

```diff
@@ -84,6 +84,12 @@ stats_writer::stats_writer(
 	: m_initialized(false), m_total_samples(0)
 {
 	m_config = config;
+	// capacity and controls should not be relevant for stats outputs, adopt capacity
+	// for completeness, but do not implement config recovery strategies.
+	if (config->m_outputs_queue_capacity > 0)
+	{
+		m_queue.set_capacity(config->m_outputs_queue_capacity);
```

Comment:
just a general question, Why did we choose to have a separate queue with respect to the outputs one?

Reply:
Also wondered about that when I added the metrics ... perhaps potential for future consolidation?